OurUmbraco/Auth/UmbracoAuthTokenDbHelper.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Linq;
+using Umbraco.Core;
+using Umbraco.Core.Persistence;
+
+namespace OurUmbraco.Auth
+{
+    public class UmbracoAuthTokenDbHelper
+    {
+        private readonly UmbracoDatabase Database = ApplicationContext.Current.DatabaseContext.Database;
```

Comment:
IMPORTANT! be sure to never do this since this means you can potentially pin a Database instance statically, for example if an instance of `UmbracoAuthTokenDbHelper` was made static. ALWAYS resolve a database instance from the DatabaseContext, never pin a database instance to a field unless you are absolutely sure that it's a transient instance.